### ISSUE TITLE
Implement the Cake build system.

### ISSUE THREAD

**Issue by RyanONeill1970** [TheGeneGenieProject/GeneGenie.Gedcom#17]
I saw a very good presentation by @gep13 recently on the [Cake](https://github.com/cake-build/cake) project, which is a C# build system.

It looks like it will take away a few issues with PowerShell and merging the list of changes, current version etc. It will also avoid us making the mistake of having build server specific scripts.

This will need to be done after #9 .

**Comment by gep13** [comment]
@RyanONeill1970 glad to hear that you like the presentation. Let me know if you have any questions about using Cake.

**Comment by M3LiNdRu** [comment]
@RyanONeill1970 I know there is #9. Meanwhile I have created and added powershell scripts and msbuild as well. Not running properly yet.

**Comment by RyanONeill1970** [comment]
OK, cool. Add the hacktoberfest label back if you want to make sure you get credit. If you want to grab this task please feel free to assign yourself and thank you.

**Comment by jeromechrist** [comment]
Sup, i can implement a basic cake build for you that you can expand on for #8 and #9

**Comment by gep13** [comment]
@jeromechrist happy to advice on this if you need any help.  I was planning on tackling it, but just haven't had the cycles to sit down and do it 😭

**Comment by RyanONeill1970** [comment]
@jeromechrist that would be good but check with @M3LiNdRu first as they've done some work on this.

**Comment by jeromechrist** [comment]
@RyanONeill1970 ok !

**Comment by RyanONeill1970** [comment]
That was quick, we'll give @M3LiNdRu a day to get back to us and then merge if no response.

Thanks @jeromechrist

**Comment by M3LiNdRu** [comment]
Good stuff!  @jeromechrist

The point is to generate the nugets during the build. Not sure, if this has been done with Cake.

Anyway, you can merge the PR and close that issue.

Thanks,
Roger

**Comment by gep13** [comment]
@M3LiNdRu yes, you can easily package nuget packages as part of your Cake Build.
